src/app/pages/FastBtcPage/containers/DepositContainer.tsx

```diff
@@ -17,16 +17,21 @@ import styles from '../fast-btc-page.module.css';
 import { SidebarStepsDeposit } from '../components/Deposit/SidebarStepsDeposit';
 import { useDepositSocket } from '../hooks/useDepositSocket';
 import { StatusScreen } from '../components/Deposit/StatusScreen';
-import { NetworkAwareComponentProps } from '../types';
+import { FastBtcDirectionType, NetworkAwareComponentProps } from '../types';
 import { Signature } from '../contexts/deposit-context';
 import { SignatureValidation } from '../components/Deposit/SignatureValidation';
 import { isMainnet } from 'utils/classifiers';
 import { Chain, ChainId } from 'types';
 import { contractReader } from 'utils/sovryn/contract-reader';
 import { debug } from 'utils/debug';
 
-export const DepositContainer: React.FC<NetworkAwareComponentProps> = ({
+type DepositContainerProps = NetworkAwareComponentProps & {
+  type: FastBtcDirectionType;
+};
+
+export const DepositContainer: React.FC<DepositContainerProps> = ({
   network,
+  type = FastBtcDirectionType.DEPOSIT,
```

Comment:
```suggestion
  type,
```
No need to provide a default value if `type` is mandatory.